CHANGELOG.md

```diff
@@ -46,12 +46,19 @@
   * Update 'use nameof instead of typeof' to support generic types (PR: [#76780](https://github.com/dotnet/roslyn/pull/76780))
   * Add feature to convert from an explicitly typed lambda to an implicitly typed one. (PR: [#76770](https://github.com/dotnet/roslyn/pull/76770))
   * Support modifiers with simple lambda parameters. (PR: [#75400](https://github.com/dotnet/roslyn/pull/75400))
-* Update Razor to 9.0.0-preview.25064.4 (PR: [#7927](https://github.com/dotnet/vscode-csharp/pull/7927))
+* Update Razor to 9.0.0-preview.25073.1 (PR: [#7940](https://github.com/dotnet/vscode-csharp/pull/7940))
   * Wire up the UseRoslynTokenizer feature properly (#11386) (PR: [#11386](https://github.com/dotnet/razor/pull/11386))
   * New Razor document formatting engine (#11364) (PR: [#11364](https://github.com/dotnet/razor/pull/11364))
   * Fix a couple of exceptions encountered when formatting documents with preprocessor directives (#11373) (PR: [#11373](https://github.com/dotnet/razor/pull/11373))
   * Allow RazorProjectEngine.Process to be cancelled (#11334) (PR: [#11334](https://github.com/dotnet/razor/pull/11334))
   * Further refactoring of Razor tooling project system (#11320) (PR: [#11320](https://github.com/dotnet/razor/pull/11320))
+  * Don't create overlapping changes when doing additional formatting (#11413) (PR: [#11413](https://github.com/dotnet/razor/pull/11413))
+  * Synchronize razor compiler assembly loading (#11394) (PR: [#11394](https://github.com/dotnet/razor/pull/11394))
+  * Allow generate method to handle delegates (#11402) (PR: [#11402](https://github.com/dotnet/razor/pull/11402))
+  * Fix bad completion commit in vs code (#11398) (PR: [#11398](https://github.com/dotnet/razor/pull/11398))
```

Comment:
🎉🎉🎉

Reply:
Oh, that issue says "There will be a corresponding PR in VS Code to actually pass the new flag.". Should I hold off merging, or remove that line item from the change log, until that happens?

Reply:
I think that is no longer true from https://github.com/dotnet/razor/pull/11398/files#r1924384742

Previously it would have needed a change to pass the flag in. The cat was away though

Reply:
Oh right. In case it wasn't clear how little attention I was paying, even though I may have appeared to be, I am glad to now cement this knowledge :)